components/contribution-flow/ContributionBlocker.js

```diff
@@ -67,6 +72,8 @@ export const getContributionBlocker = (loggedInUser, account, tier, shouldHaveTi
     return { reason: CONTRIBUTION_BLOCKER.TIER_EXPIRED, type: 'warning', showOtherWaysToContribute: true };
   } else if (account.settings.disableCustomContributions && !tier) {
     return { reason: CONTRIBUTION_BLOCKER.NO_CUSTOM_CONTRIBUTION, type: 'warning', showOtherWaysToContribute: true };
+  } else if (account.settings.disableCryptoContributions && !tier) {
+    return { reason: CONTRIBUTION_BLOCKER.NO_CRYPTO_CONTRIBUTION, type: 'warning', showOtherWaysToContribute: true };
```

Comment:
This seems wrong, as it will disable `/donate` if crypto is disabled

Reply:
Thanks, I've removed it. Don't think we need any blockers for the crypto case. 🤔 👍🏽